src/controllers/authController.js

```diff
@@ -0,0 +1,59 @@
+const { statusCode } = require("../helpers/constants.js");
+const {
+  registerUser,
+  loginUser,
+  getUserInfo,
+  addToFavorites,
+  getAllFavorites,
+  clearFavorites,
+} = require("../services/authService.js");
+
+const registerUserController = async (req, res, next) => {
+  const userInfo = req.body;
+  await registerUser(userInfo);
+  res.status(statusCode.CREATED).json({ status: "success" });
+};
+
+const loginUserController = async (req, res) => {
+  const body = req.body;
+  const token = await loginUser(body);
+  res.status(statusCode.OK).json({ status: "success", token });
+};
+
+const getUserInfoController = async (req, res) => {

```

Comment:
This is not related to auth, better to create another controller

Reply:
fixed